packages/terra-icon/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+*Changed
+  * Fixed lint error.
+
```

Comment:
CHANGELOG entries for linter changes are not necessary.
```suggestion
```

Reply:
Fixed.